packages/veritone-json-schemas/schemas/vtn-standard/master.json

```diff
@@ -0,0 +1,217 @@
+          "example": {
+            "detectionThreshold": 0.7,
+            "processingFPS": 30
+          }
+        },
+        "taskId": {
+          "type": "string",
+          "description": "<GUID> of the task (provided by Veritone)",
+          "example": "6b9cd86e-de5c-4795-b605-a9b3b29fa3cf"
+        },
+        "generatedDateUTC": {
+          "type": "string",
+          "description": "DateTime (ISO8601) of when the engine result was generated (provided by Veritone if not included)",
+          "format": "date-time",
+          "example": "2017-12-08T17:19:02Z"
+        },
+        "externalSourceId": {
+          "type": "string",
+          "description": "Vendor specific referenceId to this engine result (optional)",
+          "example": "1"
+        },
+        "validationContracts": {
+          "type": "array",
+          "items": [
+            {
+              "$ref": "#/definitions/objectResultType"
+            }
+          ]
+        }
+      }
+    },
+    "confidence": {
+      "type": "number",
+      "minimum": 0,
+      "maximum": 1,
+      "description": "Confidence of the result expressed as a range of 0.0 to 1.0"
+    },
+    "point": {
+      "type": "object",
+      "properties": {
+        "x": {
+          "type": "number",
+          "minimum": 0,
+          "maximum": 1,
+          "description": "A floating point describing the location via percentage on the x axis (1.0 would be right of the axis)"
+        },
+        "y": {
+          "type": "number",
+          "minimum": 0,
+          "maximum": 1,
+          "description": "A floating point describing the location via percentage on the y axis (1.0 would be the bottom of the axis)"
+        }
+      },
+      "required": [
+        "x",
+        "y"
+      ],
+      "description": "A point describing the 2d coordinates from the top left corner of the source."
+    },
+    "boundingPoly": {
+      "type": "array",
+      "items": {
+        "$ref": "#/definitions/point"
+      }
+    },
+    "objectCategory": {
+      "type": "array",
+      "minItems": 1,
+      "items": {
+        "type": "object",
+        "properties": {
+          "class": {
+            "type": "string",
+            "description": "The class of the object recognized",
+            "example": "For example, if the label of an object was dog, the class of the object could be animal"
+          },
+          "@id": {
+            "type": "string",
+            "description": "Private id variable used by the engine to refer to its internal dataset",
+            "example": "1"
+          }
+        },
+        "required": [
+          "class"
+        ]
+      },
+      "description": "Optional categories for object recognition. Used for referencing external taxonomies."
+    },
+    "objectResultType": {
+      "type": "string",
+      "enum": [
```

Comment:
This is still wrong.  This is the validationContract enum.  The documented valid values for object.type are:
```
      // - object: Object detection
      // - face: Face detection
      // - licensePlate: License plate detection
      // - logo: Logo detection
      // - fingerprint: Audio fingerprinting
      // - speakerId: Speaker recognition
      // - soundId: Sound recognition
      // - concept: Concept recognition
      // - keyword: Keyword detection
      // - text: Recognized or extracted text (OCR / text extraction)
      // - namedEntity: Entity extraction
      // - barcode
```